plugins/thunderbolt/fu-plugin-thunderbolt.c

```diff
@@ -379,6 +379,7 @@ fu_plugin_thunderbolt_add (FuPlugin *plugin, GUdevDevice *device)
 						     (guint) did,
 						     is_native ? "-native" : "");
 			fu_device_add_flag (dev, FWUPD_DEVICE_FLAG_UPDATABLE);
+			fu_device_add_flag (dev, FWUPD_DEVICE_FLAG_DUAL_IMAGE);
```

Comment:
is that true?

Reply:
Yep!  Tear apart an image and you'll see there are two copies of the payload.  It jumps back and forth between them when you flash.

Reply:
TIL!